packages/backend-common/src/config.ts

```diff
@@ -186,5 +218,9 @@ export async function loadBackendConfig(options: {
 
   config.setConfig(ConfigReader.fromConfigs(configs));
 
+  // Subscribe to config changes and update the redaction list for logging
+  updateRedactionMap(schema, configs, options.logger);
+  config.subscribe(() => updateRedactionMap(schema, configs, options.logger));
```

Comment:
Note to self: Probably need to get the `config` from the subscription, not the upper scope 🤔

Reply:
Nope, it's right as it is.